core/src/main/java/io/aiven/klaw/service/UsersTeamsControllerService.java

```diff
@@ -72,6 +72,7 @@ public class UsersTeamsControllerService {
   public static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
   public static final String MASKED_PWD = "********";
   public static final String UNUSED_PASSWD = "unusedpasswd";
+  private static final Pattern saasPattern = Pattern.compile("^[A-Za-z0-9+_.-]+@(.+)$");
```

Comment:
I think if we change saasPattern to just usernamePatern to avoid confusion as we removed the saas details in 2.8.0?

Reply:
or emailUsernamePattern